bin/copy_production_db_to_local

```diff
@@ -3,7 +3,7 @@
 source .env
 export RAILS_ENV=development
 
-bundle exec rails db:migrate db:drop db:create
+bundle exec rails db:drop db:create
```

Comment:
I’m not sure this change is desired - what prompted you to change it?

Reply:
By running migrate right before drop, the drop can hang b/c the database is in use. There's no reason to migrate right before dropping a database.